src/zap_hooks/requirements.txt

```diff
@@ -8,3 +8,4 @@ bleach
 selenium==4.14.0
 pyotp==2.6.0
 selenium-wire==5.1.0
+blinker==1.7.0
```

Comment:
need to update package json version even tho nothing changed... maybe try a `npm outdated` in there for a 2-for-1 sale